### PR TITLE
chore: STONEBLD-777 - tekton-ci: update bundle

### PR DESCRIPTION
**Pull request by Michkov**
Switching to new bundle since development of old one was stopped before attempt to migrate to kcp. New bundle is removing dependency on shared secret.

### CODE REVIEW

**Comment by mmalina** [comment]
@Michkov your commit message does not conform to the gitlint rules. I would suggest: `chore(STONEBLD-777): tekton-ci: update bundle`

File: .tekton/pull-request.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
       value: 'quay.io/redhat-appstudio/pull-request-builds:release-service-{{ revision }}'
   pipelineRef:
     name: docker-build
-    bundle: quay.io/redhat-appstudio/hacbs-core-service-templates-bundle:latest
+    bundle: quay.io/redhat-appstudio-tekton-catalog/pipeline-core-services-docker-build:latest
```
Comment (mmalina) [inline .tekton/pull-request.yaml:20]: What do you think of tying this to a specific version? Not saying it's needed, just something to consider. (Same thing in the push pipeline.)

Comment (Michkov) [reply .tekton/pull-request.yaml:20]: the idea here was to be customer one of the pipelines, so rather fail for us than for customer.

Comment (mmalina) [reply .tekton/pull-request.yaml:20]: Alright, fair point.

**Review by mmalina — APPROVED** [review]
LGTM

**Pull request merged**